source/common/grpc/rpc_channel_impl.cc

```diff
@@ -124,9 +124,6 @@ void RpcChannelImpl::onFailure(Http::AsyncClient::FailureReason reason) {
   case Http::AsyncClient::FailureReason::Reset:
```

Comment:
I went back and forth on this. Since the enum is already there and plumbed everywhere, it would be a lot of work to change it to no parameters, then we might need a parameter again. I'm inclined to leave it for now?